src/DatePicker/DatePickerDialog.js

```diff
@@ -136,7 +138,7 @@ class DatePickerDialog extends Component {
           repositionOnUpdate={true}
           open={open}
           onRequestClose={this.handleRequestClose}
-          style={styles.dialogBodyContent}
+          style={Object.assign(styles.dialogBodyContent, containerStyle)}
```

Comment:
@oliviertassinari I've updated the PR to remove the duplicate `{...other}` as mentioned by @mbrookes. Anything else?